extensions/arc/src/ui/dialogs/addPGExtensionsDialog.ts

```diff
@@ -17,6 +17,8 @@ export class AddPGExtensionsDialog extends InitializingComponent {
 
 	protected _completionPromise = new Deferred<string | undefined>();
 
+	private validExtensions = ['citus', 'pgaudit', 'pgautofailover', 'pg_cron', 'pg_partman', 'plv8', 'postgis', 'postgis_raster', 'postgis_sfcgal', 'postgis_tiger_geocoder', 'tdigest'];
```

Comment:
Since this never changes I'd suggest just having this be a const outside the class.